src/constants/contracts.ts

```diff
@@ -52,8 +52,9 @@ import vtrx from 'assets/img/coins/vtrx.png';
 import vust from 'assets/img/coins/vust.png';
 import vluna from 'assets/img/coins/vluna.png';
 
+import { TokenSymbol } from 'types';
 import { isOnTestnet } from 'config';
-import contracts from './contracts';
+import contracts from 'utilities/contracts';
 
 export const CONTRACT_XVS_TOKEN_ADDRESS = isOnTestnet
```

Comment:
I think that name is confusing. It isn't just the address but the actual token. Perhaps `XVS_TOKEN`.